base/funknown.cpp

```diff
@@ -84,7 +84,7 @@ namespace FUnknownPrivate {
 int32 PLUGIN_API atomicAdd (int32& var, int32 d)
 {
 #if SMTG_OS_WINDOWS
-	return InterlockedExchangeAdd (&var, d) + d;
+	return InterlockedExchangeAdd ((LONG*)&var, d) + d;
```

Comment:
I disagree with that.
Use atomic from the c++ standard library.

Reply:
When trying to include `<stdatomic.h>` as do Android and Mac, it complains about `_Atomic` not defined. Including `<atomic>` worked fine though.
Do you suggest that I cast it into a `std::atomic<int32>*` ?

Reply:
No, I suggest to rewrite it all to use std::atomic<int> instead.
Cast are absolutely not needed here.
But check with the Steinberg guys what to do, I am not working at Steinberg.